pkg/snapstore/gcs_snapstore.go

```diff
@@ -50,6 +52,12 @@ type GCSSnapStore struct {
 	maxParallelChunkUploads uint
 	minChunkSize            int64
 	tempDir                 string
+	chunkDirSuffix          string
+}
+
+type EmulatorConfig struct {
```

Comment:
Since `EmulatorConfig` is only used in `gcs_snapstore.go`, is there any reason to make this struct exported in `package snapstore`?